File: example/java/rxjava-apis-example-client/src/main/java/org/rxjava/api/client/example/TestApi.java
package org.rxjava.api.client.example;

import reactor.core.publisher.Mono;

import java.util.*;
import java.util.Map.Entry;
import java.util.concurrent.Future;

import org.rxjava.apikit.client.*;

import org.rxjava.api.client.example.form.TestForm;

public class TestApi {
	private ClientAdapter clientAdapter;

	public TestApi() {
	}

	public TestApi(ClientAdapter clientAdapter) {
		this.clientAdapter = clientAdapter;
	}

	public Mono<Integer> testewwPath(String id, TestForm form) {
		Map<String, Object> _uriVariables = new HashMap<>();
		_uriVariables.put("id", id);
		String _url = ApiUtils.expandUriComponent("person/testeeee/{id}", _uriVariables);

		List<Entry<String, Object>> _form = form.encode("", new ArrayList<>());
		return clientAdapter.request("GET", _url, _form, _0Type);
	}

	public void setclientAdapter(ClientAdapter clientAdapter) {
		this.clientAdapter = clientAdapter;
	}

	public ClientAdapter getclientAdapter() {
		return clientAdapter;
	}

	private static final ApiType _0Type = ApiUtils.type(Integer.class);
}

File: example/js/rxjava-apis-example-client/TestApi.js
import TestForm from './form/TestForm'
import {AbstractApi} from 'rxjava-api-core'


class TestApi extends AbstractApi {
        
    /**
        * 
     *
        * <div class='http-info'>http 说明<ul>
    * <li><b>Uri:</b>person/testeeee/{id}</li>
    * <li><b>PathVariable:</b> string id</li>
    * <li><b>Form:</b>TestFormtestewwPath</li>
    * <li><b>Model:</b> number</li>
    * <li>需要登录</li>
    * </ul>
    * </div>
    * @see TestForm
     */
    testewwPath = (id, form) => {
        let _path = {};
        _path["id"] = id;
        return super._request("example", "testApi", "GET", "person/testeeee/{id}", _path, form);
    }
}

export { TestApi };
declare const testApi: TestApi;
export default testApi;